src/connectedk8s/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 
+1.3.21
+++++++
+
+* Setting default authnMode to 'arc' when RBAC is enabled
```

Comment:
`Setting guard default authnMode to `arc`....`

Reply:
Updated here: https://github.com/Azure/azure-cli-extensions/pull/6675